SpatialGDK/Source/SpatialGDK/Public/Utils/ComponentFactory.h

```diff
@@ -29,18 +31,18 @@ class SPATIALGDK_API ComponentFactory
 public:
 	ComponentFactory(bool bInterestDirty, USpatialNetDriver* InNetDriver);
```

Comment:
What about passing the Tracer into the factory itself, and storing it as a member?